### ISSUE TITLE
Is it possible to control the Mi Smart STanding fan 2 Lite (dmaker.fan.1c) using a Python script?

### ISSUE THREAD

**Issue by romulovieira-me** [syssi/xiaomi_fan#234]
Hello,

I have a Mi Smart Standing fan 2 Lite and I can control it using the CLI of the python-miio library (https://python-miio.readthedocs.io/en/latest/).
However, I would like to do this using the modules from this library in a Python script. Is It possible?

Thank you for your help.

Best Regards

**Comment by romulovieira-me** [comment]
Hello,

I have a Mi Smart STanding fan 2 Lite and I can control it using the CLI of the python-miio library (https://python-miio.readthedocs.io/en/latest/).
However, would I like to do this using the modules from this library in a Python script? It is possible?

Thank you for your help.

Best Regards

**Comment by syssi** [comment]
Yes, this project is well suited to your use case: https://github.com/rytilahti/python-miio/
